x-pack/test/api_integration/apis/ml/saved_objects/sync.ts

```diff
@@ -45,6 +57,14 @@ export default ({ getService }: FtrProviderContext) => {
     });
 
     it('should sync datafeeds and saved objects', async () => {
+      // check to see is a sync is needed
```

Comment:
Nit: typo, that is copy/pasted a couple times in this file
```suggestion
      // check to see if a sync is needed
```